### PR TITLE
Evolution script to set digital_leben workflow to 'participate' phase

### PR DESCRIPTION
**Pull request by pallix**

### CODE REVIEW

File: src/adhocracy_spd/adhocracy_spd/evolution/__init__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+"""Scripts to migrate legacy objects in existing databases."""
```
Comment (nidico) [inline src/adhocracy_spd/adhocracy_spd/evolution/__init__.py]: Docstring doesn't match content.

**Comment by nidico** [comment]
I'd prefer to have this as a command line script, as - I assume - we don't want to evolve the production system to `participate` immediately.

**Comment by pallix** [comment]
Having a command line script is probably a good idea, then developers can also play locally with different phases while developping.

**Comment by nidico** [comment]
Is this obsolete now after merging #1340?

**Comment by pallix** [comment]
Yep.

**Pull request closed without merging**